src/pages/dataset-registration-page/form-informationmodel/fdk-information-models/fdk-information-models.component.jsx

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Autosuggest from 'react-autosuggest';
+import _ from 'lodash';
+import {
+  extractInformationModels,
+  searchInformationModels
+} from '../../../../services/api/search-api/information-models';
+import localization from '../../../../services/localization';
+import { getTranslateText } from '../../../../services/translateText';
+import { datasetFormPatchThunk } from '../../formsLib/asyncValidateDatasetInvokePatch';
+import { getConfig } from '../../../../config';
+import { informationModelType } from '../../../../schemaTypes';
+
+import '../form-informationmodel.component.scss';
+
+class FdkInformationModelsSuggestionField extends React.Component {
```

Comment:
Det er best å bruke hooks for å ikke bryte måten resten av komponenter er skrevet i applikasjonen

Reply:
1b53e25

ok?